docs/api-irma-server.md

```diff
@@ -250,7 +250,7 @@ endpoint can only be used when the [session status](#get-irma-session-clienttoke
 The body of an options request should have the following structure:
 ```json
 {
-  "@context": "https://irma.app/ld/request/options/v1",
+  "@context": "https://irma.app/ld/request/frontendoptions/v1",
```

Comment:
After I approved it, I saw that there is another occurrence of this context string below. Maybe you can fix it there too?

Reply:
That one is correct.

Reply:
Thanks!